modules/installation/proc-rhdh-deploy-eks-operator.adoc

```diff
@@ -1,12 +1,20 @@
-[id='proc-rhdh-deploy-eks-using-operator_{context}']
+// Module included in the following assemblies
+// assembly-install-rhdh-eks.adoc
 
-= Deploying {product} on {eks-name} ({eks-short}) using the Operator
+[id='proc-rhdh-deploy-eks-operator_{context}']
+= Installing {product-short} on {eks-short} with the Operator
```

Comment:
Using the full product name in the headings ensures consistency and is also beneficial for SEO.
```suggestion
= Installing {product} on {eks-short} with the Operator
```

Reply:
I do see that there is some inconsistencies with the headings in this file because they are not modularized, so I will change the product name in the sub-headings. 

https://issues.redhat.com/browse/RHIDP-3290 modularizes this content in the future